### ISSUE TITLE
paddle.metric.Auc() api样例描述不够详细

### ISSUE THREAD

**Issue by Li-fAngyU** [PaddlePaddle/Paddle#38053]
import paddle
from sklearn.metrics import roc_auc_score

auc = paddle.metric.Auc()
preds = np.concatenate( (output,1-output),axis = 1 )
auc.update(labels = label,preds = preds)
print(auc.accumulate())
输出：0.45912389292350253

auc = paddle.metric.Auc()
preds = np.concatenate( (1-output,output),axis = 1 )
auc.update(labels = label,preds = preds)
auc.accumulate()
输出：0.5408761070764974

roc_auc_score(y_true = label,y_score = output)
输出：0.5408555920917664

问题在于preds 当concatenate的顺序互换时结果会有差异，希望能改进下像sklearn一样只需要输入单一预测就好了。

**Comment by qili93** [comment]
收到，会作为新需求进行评估，感谢您的反馈。
